openshift/knative-eventing-encryption-auth.yaml

```
@@ -0,0 +1,5 @@
+spec:
+  config:
+    config-features:
+      transport-encryption: "strict"
+      authentication.oidc: "enabled"
```

```suggestion
      authentication-oidc: "enabled"
```
Can we update this here already (https://github.com/openshift-knative/eventing/pull/536)